python/eigen.cpp

```diff
@@ -16,6 +16,8 @@ void register_eigen(py::module &m) {
                       [](Color &c, float v) { c.g() = v; }, D(Color, g))
         .def_property("b", [](const Color &c) { return c.b(); },
                       [](Color &c, float v) { c.b() = v; }, D(Color, b))
+        .def_property("w", [](const Color &c) { return c.w(); },
```

Comment:
This one needs to be bound with an explicit docstring because `make mkdoc` will not get the inherited Eigen docs (which is where `Vector4f::w()` comes from).  This is used in the new examples.